diff --git a/drivers/amlogic/gpu/mali/platform/meson_m450/platform_m8.c b/drivers/amlogic/gpu/mali/platform/meson_m450/platform_m8.c
index 62f2aad..81a6d92 100755
--- a/drivers/amlogic/gpu/mali/platform/meson_m450/platform_m8.c
+++ b/drivers/amlogic/gpu/mali/platform/meson_m450/platform_m8.c
@@ -228,7 +228,7 @@ void meson_platform_get_clock_info(struct mali_gpu_clock **data) {
 
 /* Function that get the current clock info, needed when CONFIG_MALI_DVFS enabled */
 int meson_platform_get_freq(void) {
-    printk("get cur_gpu_clk_index =%d\n", cur_gpu_clk_index);
+    //printk("get cur_gpu_clk_index =%d\n", cur_gpu_clk_index);
     return  cur_gpu_clk_index;
 }
 
